unliftio/package.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - deepseq
   - directory
   - filepath
+  - nats
```

Comment:
-1, I'd rather avoid adding this dependency. Is this only needed for the `Natural` type for older versions of GHC? I'd feel more comfortable about making it a conditional dependency then.